R/check-cran.sh

```diff
@@ -20,14 +20,14 @@
 set -o pipefail
 set -e
 
-FWDIR="$(cd `dirname "${BASH_SOURCE[0]}"`; pwd)"
-pushd $FWDIR > /dev/null
+FWDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")"; pwd)"
```

Comment:
Ah OK, it does seem to work. There are other instances below though that I suppose should be consistent